cmd/nomos/status/cluster_state.go

```diff
@@ -55,7 +55,9 @@ func (c *ClusterState) printRows(writer io.Writer) {
 	}
 	for _, repo := range c.repos {
 		fmt.Fprintf(writer, "%s%s\n", util.Indent, util.Separator)
-		repo.printRows(writer)
+		if len(rsyncname) <= 0 || (len(rsyncname) > 1 && repo.syncName == rsyncname) {
```

Comment:
`if rsyncname == "" || rsyncname == repo.syncName`

Reply:
thanks, i have updated to follow your sample